cli/src/pcluster/templates/cw_dashboard_builder.py

```diff
@@ -172,15 +179,29 @@ def _add_text_widget(self, markdown):
         self.cloudwatch_dashboard.add_widgets(text_widget)
         self._update_coord_after_section(d_y=1)
 
-    def _generate_graph_widget(self, title, metric_list):
+    def _generate_graph_widget(self, title, metric_list, error_label):
```

Comment:
Yes, set the default error_label to false since in most if the case it is false. I updated the PR to add a todo in the code on whether we set the annotation to True or add widgets below the graph widgets to show the troubleshooting links. Will update the PR once the decision is made